### DISCUSSION TITLE
Feature request: option to display time info of the panel as absolute time range

### DISCUSSION THREAD

**Discussion by rcasta74** [grafana/grafana#45944] [Dashboards, Panels & Visualizations]
I'm using a gauge bar panel with a variable defined as `0d,1d,2d,3d,...`, used both in _Repeat options_ and _Query option_ (as _Time shift_ parameter). In this way I can compare the data belonging to different days within the same time interval (for example: from 02:00 to 08:00).
The time info for each panel is displayed like _timeshift -1d_; it would be more confortable if it could be displayed as absolute time interval instead (for example like _2022-02-25 02:00:00 to 2022-02-25 07:59:59_).
An alternative would be to have new variables such as `$__from` and `$__to` but which refer to the relative time interval of the panel, so that they can be used for the title, as was somehow requested in #38954

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
